cmp/grid/GridModel.js

```diff
@@ -79,7 +79,7 @@ export class GridModel {
     groupRowRenderer;
     /** @member {Grid~groupRowElementRendererFn} */
     groupRowElementRenderer;
-    /** @member {GridGroupSortFn} */
+    /** @member {(null|GridGroupSortFn)} */
```

Comment:
Don't think we have been generally calling out nullable values in this way.  Can we skip?  And also put the description of what null does last on line 165

Reply:
Hi,  I have the suggested changes to the docs.